getting-started/deployment/aws-ec2.mdx

```diff
@@ -87,11 +87,100 @@ After making changes, restart the Docker Compose containers:
+For production usage, it's recommended to enable TLS (HTTPS). Without TLS, cloud templates may not load properly when using the Safari browser.
 
-Without TLS, cloud templates may not load properly when using the Safari browser.
+To secure OpenOps with TLS, it's recommended to place your EC2 instance behind an AWS Application Load Balancer (ALB). This setup uses AWS Certificate Manager (ACM) to handle TLS, and allows secure access via a DNS name, even without a [custom domain](#accessing-openops-via-a-custom-domain-over-tls).
+
+<Warning>Once the ALB is in place, all access should go through it — you can no longer use the EC2 public IP directly.</Warning>
+
+### Request a TLS certificate in ACM
+
+1. Go to the [AWS Certificate Manager](https://console.aws.amazon.com/acm/home) (ACM).
+2. Request a public certificate for your EC2's public DNS name (e.g., `ec2-1-2-3-4.compute-1.amazonaws.com`).
+3. Choose **DNS validation** and wait for the certificate to be **Issued**.
+
+<Tip>Assign an elastic IP to your EC2 instance to prevent its IP from changing over time.</Tip>
+
+### Create an Application Load Balancer
+
+1. In the **EC2 > Load Balancers** section, click **Create Load Balancer**.
+2. Choose **Application Load Balancer**, and configure the following:
+    * **Scheme**: Internet-facing
+    * **Listeners**:
+        * Add a listener for **HTTPS (443)** using the ACM certificate.
+        * Optionally, add a listener on port **80** with a rule to redirect HTTP traffic to HTTPS.
+3. Under **Security Groups**, create a security group that allows inbound traffic on **port 443**.
+4. Create a target group:
+    * **Target type**: Instance
+    * **Protocol**: HTTP
+    * **Port**: 80
+    * **Health check path**: `/api/v1/health`
+5. Register your EC2 instance as a target.
+6. Finish creating the ALB and wait for its status to become **Active**.
+
+Once setup is complete, you'll be able to access OpenOps securely at `https://your-alb-dns-name.elb.amazonaws.com`.
+
+### Update environment and restart
+
+1. SSH into your EC2 instance or use CloudShell.
+2. Update the public OpenOps URL in the `.env` file:
+    ```env
+    OPS_PUBLIC_URL=https://your-alb-dns-name.elb.amazonaws.com
+    ```
+3. Restart the containers:
+    ```bash
+    sudo docker compose down && sudo docker compose up -d
+    ```
+
+### Troubleshooting
+
+If you encounter a 503 error:
+* Ensure you restarted Docker containers after updating `.env`.
+* Confirm that the ALB's health check is configured with the correct path (`/api/v1/health`).
+* Make sure the EC2 instance's security group allows traffic from the ALB.
+* Look for more cues in the Nginx container logs:
+    ```bash
+    sudo docker compose logs nginx
+    ```
+
+For more troubleshooting guidance, see [this AWS troubleshooting guide](https://repost.aws/knowledge-center/alb-troubleshoot-503-errors).
+
+## Accessing OpenOps via a custom domain over TLS
```

Comment:
Sorry for misleading you, the custom domain mapping and TLS should stay together, and not be split into separate guides. I only meant that the title should reflect that.

Reply:
Reverted to the original version that only contains one section and clarified the title. Please check again